### DISCUSSION TITLE
v2.2.5

### DISCUSSION THREAD

**Discussion by Zenda-Cross** [vega-org/vega-app#68] [Announcements]
- New source Primewire
- Bug  fixes #64  #57
- Providers Improved.

<hr /><em>This discussion was created from the release <a href='https://github.com/Zenda-Cross/vega-app/releases/tag/v2.2.5'>v2.2.5</a>.</em>

**Comment by imsuchitrajena** [comment]
You Should Induct More Providers..
Sometimes HQ Flix Isn't Working & More Hollywood Movies Or International Movies Providers Are Less Than Bollywood & South Industry

**Reply by Zenda-Cross** [reply]
certainly, I will try.
